tests/src/test/scala/runtime/actionContainers/ActionLoopGoContainerTests.scala

```diff
@@ -135,4 +135,28 @@ abstract class ActionLoopGoContainerTests
       c.run(helloMsg()) should be(okMsg("hello-Hello", "Hello, Demo!"))
     }
   }
+
+  it should "support return array result" in {
+    val helloArrayGo = {
+      s"""
+         |package main
+         |
+         |func Main(obj map[string]interface{}) []interface{} {
```

Comment:
So if we want to use the JSON array params, do we have to use this signature for the main function?

Reply:
for JSON array params, can use like blow
```
// Main is the function implementing the action
func Main(event []interface{}) []interface{} {
	input := make([]string, len(event))
	for i, v := range event {
		input[i] = fmt.Sprint(v)
	}
	sort.Strings(input)
	output := make([]interface{}, len(event))
	for i, v := range input {
		output[i] = v
	}
	return output
}
```

hm..it is better to add a test case for json array params as well.

Reply:
ok now I get it.

So originally the signature of the `Main` method is 

```
func Main(obj map[string]interface{}) map[string]interface{} {
...
}
```

And now we add support for JSON array params.